README.md

```diff
@@ -0,0 +1,14 @@
+# Building your layout from source
```

Comment:
Not sure why this file was added? QMK already has a readme.

Reply:
Mistake, was going through some generated layouts